### PR TITLE
[OSDOCS-10074]:Expose cluster delete protection feature (ROSA and OSD UI)

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.15+
Issue:

https://issues.redhat.com/browse/OSDOCS-10074
Link to docs preview:

**GCP on OSD**: https://75739--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_install_access_delete_cluster/creating-a-gcp-cluster.html#osd-create-gcp-cluster-ccs_osd-creating-a-cluster-on-gcp
**Reviewers: Please note that this new optional step is included (duplicated) in the highlighted subtopics:** 
image
**AWS on OSD**: https://75739--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_install_access_delete_cluster/creating-an-aws-cluster.html#osd-create-aws-cluster-ccs_osd-creating-a-cluster-on-aws

**ROSA**: https://75739--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_getting_started/rosa-quickstart-guide-ui.html#rosa-quickstart-creating-a-cluster
**ROSA**: https://75739--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.html#rosa-sts-creating-a-cluster-using-defaults-ocm_rosa-sts-creating-a-cluster-quickly

QE review:
- [x] QE has approved this change.

Additional information:

**DO NOT MERGE UNTIL FEATURE RELEASE. SCHEDULED FOR 05/16.**

### CODE REVIEW

**Comment by mletalie** [comment]
Peer review squad lucky recipient :) 
As an FYI, this feature is still in staging; however, it is slated to go into production this week. QA has been notified for review request, but would like to get a preliminary peer review now  so as to be as ready as possible when this does go live. FWIW all though there are several topics "touched", the update is the same throughout (ie added to relevant topics). Thanks!

File: modules/osd-create-cluster-rhm-gcp-account.adoc [hunk 1]
```diff
@@ -177,6 +177,10 @@ In the event of critical security concerns that significantly impact the securit
 ====
 
 . Review the summary of your selections and click *Create cluster* to start the cluster installation. The installation takes approximately 30-40 minutes to complete.
++
+. Optional: Within the *Overview* tab, you can enable the delete protection feature by selecting 'Enable', which is located directly under *Delete Protection: Disabled*. This will prevent your cluster from being deleted. To disable delete protection, select 'Disable'.
```
Comment (bhardesty) [inline modules/osd-create-cluster-rhm-gcp-account.adoc:181]: Same comment as above.

File: modules/rosa-sts-creating-a-cluster-using-defaults-ocm.adoc [hunk 2]
```diff
@@ -47,11 +47,15 @@ If your AWS account ID is not listed, check that you have successfully associate
 +
 ====
 [NOTE]
-Cluster creation generates a domain prefix as a subdomain for your provisioned cluster on `openshiftapps.com`. If the cluster name is less than or equal to 15 characters, that name is used for the domain prefix. If the cluster name is longer than 15 characters, the domain prefix is randomly generated to a 15 character string. To customize the subdomain, select the *Create custom domain prefix* checkbox, and enter your domain prefix name in the *Domain prefix* field. 
+Cluster creation generates a domain prefix as a subdomain for your provisioned cluster on `openshiftapps.com`. If the cluster name is less than or equal to 15 characters, that name is used for the domain prefix. If the cluster name is longer than 15 characters, the domain prefix is randomly generated to a 15 character string. To customize the subdomain, select the *Create custom domain prefix* checkbox, and enter your domain prefix name in the *Domain prefix* field.
```
Comment (bhardesty) [inline modules/rosa-sts-creating-a-cluster-using-defaults-ocm.adoc:50]: Same comment as above.

File: modules/rosa-sts-creating-a-cluster-using-defaults-ocm.adoc [hunk 3]
```diff
@@ -47,11 +47,15 @@ If your AWS account ID is not listed, check that you have successfully associate
...
 ====
 . To deploy a cluster quickly, leave the default options in the *Cluster settings*, *Networking*, *Cluster roles and policies*, and *Cluster updates* pages and click *Next* on each page.
 
 . On the *Review your ROSA cluster* page, review the summary of your selections and click *Create cluster* to start the installation.
++
+. Optional: Within the *Overview* tab, you can enable the delete protection feature by selecting 'Enable', which is located directly under *Delete Protection: Disabled*. This will prevent your cluster from being deleted. To disable delete protection, select 'Disable'.
```
Comment (bhardesty) [inline modules/rosa-sts-creating-a-cluster-using-defaults-ocm.adoc:56]: Same comment as above.

**Review by bhardesty — COMMENTED** [review]
Looks good! I just had a couple very minor grammatical comments. Since this PR adds the same text to multiple files, you'll need to make the changes in each of those places.

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Pull request merged**